Src/BootCamp.Chapter/Shop.cs

```diff
@@ -1,28 +1,21 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Shop
     {
-        private decimal _money;
-        public decimal GetMoney()
-        {
-            return _money;
-        }
-
-        private Inventory _inventory;
+        public Inventory Inventory { get; private set; }
```

Comment:
I had to expose the inventory component through a public getter so it can be used inside ShopTests.cs.

Reply:
That's alright. It's a problem of a test, but it's okay.